packages/@okta/vuepress-site/docs/reference/okta-expression-language-in-identity-engine/index.md

```diff
@@ -253,3 +253,23 @@ If the middle initial is not empty, include it as part of the full name, using j
 
 If the user is a contractor and is a member of the "West Coast Users" user group, output "West coast contractors", else output "Others".<br>
 `user.profile.isContractor && user.isMemberOf({'group.profile.name': 'West Coast Users'}) ? "West coast contractors" : "Others"`
+
+## Security Expressions
+
+In [Okta Sign On Policies](/docs/reference/api/policy/#okta-sign-on-policy) and [App Sign On Policies](/docs/reference/api/policy/#app-sign-on-policy), you can additionally specify certain [Rule Conditions](/docs/reference/api/policy/#conditions) using expressions based on the security context of the request.
```

Comment:
Only app-sign-on policies support EL for OIE orgs, right? I didnt know there are some EL supported for Okta sign-on.

Reply:
Good point.  Removed the reference to Okta Sign On Policies.